### PR TITLE
docs(sample): add missing samples

### PR DESCRIPTION
**Pull request by laljikanjareeya**
PR contains following samples:

- storage_object_csek_to_cmek
- storage_compose_file
- storage_download_public_file
- storage_get_service_account
- storage_set_bucket_public_iam
- storage_bucket_delete_default_kms_key
- storage_define_bucket_website_configuration

### CODE REVIEW

File: samples/system-test/encryption.test.js [hunk 1]
```diff
@@ -87,3 +88,19 @@ it('should rotate keys', () => {
...
   assert.include(output, 'Encryption key rotated successfully.');
 });
+
+it('should convert CSEK to KMS key', async () => {
+  const encryptedFileName = 'encrypted-file';
+  await bucket
+    .file(encryptedFileName, {
+      encryptionKey: key,
+    })
+    .save();
```
Comment (AVaksman) [inline samples/system-test/encryption.test.js:98]:
```suggestion
    .save('test);
```
I believe `data` is a required param for `file#save()`

**Review by AVaksman — CHANGES_REQUESTED** [review]

File: samples/system-test/encryption.test.js (hunk 1 above)
Comment (laljikanjareeya) [reply samples/system-test/encryption.test.js]: Done !!

File: samples/changeFileCSEKToCMEK.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+// sample-metadata:
+//   title: Storage File Convert CSEK to CMEK.
+//   description: Storage File Convert CSEK to CMEK.
+//   usage: node changeFileCSEKToCMEK.js <BUCKET_NAME> <FILE_NAME> <ENCRYPTION_KEY> <KMS_KEY_NAME>
+
+function main(
+  bucketName = 'my-bucket',
+  fileName = 'test.txt',
+  encryptionKey = 'TIbv/fjexq+VmtXzAlc63J4z5kFmWJ6NdAPQulQBT7g=',
+  kmsKeyName = 'projects/your-project-id/locations/global/keyRings/your-key-ring/cryptoKeys/your-key'
```
Comment (AVaksman) [inline samples/changeFileCSEKToCMEK.js:26]: should it be set to `process.env.GOOGLE_CLOUD_KMS_KEY_US`?

File: samples/changeFileCSEKToCMEK.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+// sample-metadata:
+//   title: Storage File Convert CSEK to CMEK.
+//   description: Storage File Convert CSEK to CMEK.
+//   usage: node changeFileCSEKToCMEK.js <BUCKET_NAME> <FILE_NAME> <ENCRYPTION_KEY> <KMS_KEY_NAME>
+
+function main(
+  bucketName = 'my-bucket',
+  fileName = 'test.txt',
+  encryptionKey = 'TIbv/fjexq+VmtXzAlc63J4z5kFmWJ6NdAPQulQBT7g=',
```
Comment (AVaksman) [inline samples/changeFileCSEKToCMEK.js:25]: should it be set to as example?  `'my-encription-key'`?

**Review by AVaksman — APPROVED** [review]

**Comment by laljikanjareeya** [comment]
Gentle ping @JesseLovelace

File: samples/changeFileCSEKToCMEK.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+// sample-metadata:
+//   title: Storage File Convert CSEK to CMEK.
+//   description: Storage File Convert CSEK to CMEK.
+//   usage: node changeFileCSEKToCMEK.js <BUCKET_NAME> <FILE_NAME> <ENCRYPTION_KEY> <KMS_KEY_NAME>
+
+function main(
+  bucketName = 'my-bucket',
+  fileName = 'test.txt',
+  encryptionKey = 'my-encription-key',
+  kmsKeyName = process.env.GOOGLE_CLOUD_KMS_KEY_US
```
Comment (frankyn) [inline samples/changeFileCSEKToCMEK.js:26]: Remove the call to get an environment variable `GOOGLE_CLOUD_KMS_KEY_US`.

File: samples/downloadPublicFile.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+//   usage: node downloadPublicFile.js <BUCKET_NAME> <SRC_FILE_NAME> <DEST_FILE_NAME>
+
+const path = require('path');
+const cwd = path.join(__dirname, '..');
+
+function main(
+  bucketName = 'my-bucket',
+  srcFileName = 'test.txt',
+  destFileName = path.join(cwd, 'downloaded.txt')
+) {
```
Comment (frankyn) [inline samples/downloadPublicFile.js:29]: This sample should use an unathenticated client instead of http package directly.

**Review by frankyn — DISMISSED** [review]

File: samples/downloadPublicFile.js (hunk 5 above)
Comment (frankyn) [reply samples/downloadPublicFile.js:29]: https://github.com/googleapis/nodejs-storage/pull/82/files

Comment (laljikanjareeya) [reply samples/downloadPublicFile.js:29]: we don't have unauthenticated client like in [JAVA](https://github.com/googleapis/google-cloud-java/blob/5203d14fbc3af58a9cae2838b908fec42ddf5f8d/google-cloud-examples/src/main/java/com/google/cloud/examples/storage/objects/DownloadPublicObject.java#L38) I just modified the sample that explain how to remove authentication.

File: samples/addBucketWebsiteConfiguration.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+  async function addBucketWebsiteConfiguration() {
+    await storage.bucket(bucketName).setMetadata({
+      website: {
+        mainPageSuffix,
+        notFoundPage,
+      },
+    });
+
+    console.log(`Website configuration has been added to ${bucketName}.`);
```
Comment (JesseLovelace) [inline samples/addBucketWebsiteConfiguration.js:49]: Make this more descriptive, specifically say what pages were set and to what values. See the java sample's output for guidance (https://cloud.google.com/storage/docs/hosting-static-website-http#storage-set-website-config-java)

Comment (laljikanjareeya) [reply samples/addBucketWebsiteConfiguration.js]: Done!

File: samples/downloadPublicFile.js [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+  // [START storage_download_public_file]
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const bucketName = 'Name of a bucket, e.g. my-bucket';
+  // const srcFileName = 'Remote file to download, e.g. file.txt';
+  // const destFileName = 'Local destination for file, e.g. ./local/path/to/file.txt';
+
+  // CI authentication is done with ADC. Cache it here, restore it after file downloaded.
+  const GOOGLE_APPLICATION_CREDENTIALS =
```
Comment (frankyn) [inline samples/downloadPublicFile.js:39]: Is there no way to pass in a config with no credentials?

Working with environment variables is definitely something we shouldn't do.

File: samples/downloadPublicFile.js (hunk 5 above)
Comment (frankyn) [reply samples/downloadPublicFile.js:29]: @stephenplusplus is this a feature gap in the library?

**Review by JesseLovelace — APPROVED** [review]

**Review by shaffeeullah — APPROVED** [review]

**Pull request merged**